lib/facilities/ppms_configuration.rb

```diff
@@ -6,14 +6,14 @@
 
 module Facilities
   class PPMSConfiguration < Common::Client::Configuration::REST
-    self.open_timeout = 1500
-    self.read_timeout = 1500
+    self.open_timeout = 60
```

Comment:
Could this be a [Settings](https://github.com/department-of-veterans-affairs/vets-api/blob/017627a84721e8dfe4e2cf6f99cfaa7f72a4f000/lib/evss/letters/configuration.rb#L6) var so it can be tweaked per environment? Also I believe the forward proxy has a timeout of `60` seconds and if it won the race you'd get a different error than `GatewayTimeout`. In the letters example above we set it to `55`.